blueprints/component-test/files/tests/unit/__path__/__test__.coffee

```diff
@@ -5,13 +5,13 @@ moduleForComponent '<%= dasherizedModuleName %>', '<%= classifiedModuleName %>Co
   # needs: ['component:foo', 'helper:bar']
 }
 
-test 'it renders', ->
-  expect 2
+test 'it renders', (assert) ->
+  assert.expect 2
 
   # creates the component instance
   component = @subject()
-  equal component._state, 'preRender'
+  assert.equal component._state, 'preRender'
 
   # appends the component to the page
   @append()
```

Comment:
This has been renamed to `render()` (https://github.com/ember-cli/ember-cli/commit/b13bde07128680180baf96398de496bc7a7e7c8d)